client/pages/ProfilePage.tsx

```diff
@@ -1,10 +1,89 @@
-import { Text, TouchableOpacity, View } from "react-native";
+import { Alert, Text, TouchableOpacity, View } from "react-native";
 import sharedStyles from "../src/styles/shared";
-import { useStytch, useStytchUser } from "@stytch/react-native-expo";
+import { useStytch, useStytchUser } from "@stytch/react-native-expo-testing";
+import React, { useState, useEffect } from "react";
 
 function ProfilePage() {
   const stytch = useStytch();
   const user = useStytchUser();
+
+  const [isKeystoreAvailable, setKeystoreAvailable] = useState(false);
+  const [hasBiometricRegistration, setBiometricRegistration] = useState(false);
+  const [sensorType, setSensorType] = useState("none");
+
+  const checkKeystoreAvailable = () => {
+    stytch.biometrics.isKeystoreAvailable().then((resp) => {
+      setKeystoreAvailable(resp);
+    });
+  };
+
+  const checkBiometricRegistration = () => {
+    stytch.biometrics.isRegistrationAvailable().then((resp) => {
+      setBiometricRegistration(resp);
+    });
+  };
+
+  const checkSensor = () => {
+    stytch.biometrics
+      .getSensor()
+      .then((resp) => {
+        const { biometryType } = resp;
+        setSensorType(biometryType);
+      })
+      .catch((err) => console.log(err));
+  };
+
+  useEffect(() => {
```

Comment:
must have a dependency array. Could probably be `[]` or `[stytch]` and could wrap the functions in an `if (stytch)`